migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc

```diff
@@ -20,9 +24,15 @@ You can configure a migration plan to suit your needs by increasing the number o
 include::modules/migration-changing-migration-plan-limits.adoc[leveloffset=+2]
 include::modules/migration-excluding-resources.adoc[leveloffset=+2]
 
-include::modules/migration-adding-cluster-to-cam.adoc[leveloffset=+1]
-include::modules/migration-adding-replication-repository-to-cam.adoc[leveloffset=+1]
-include::modules/migration-creating-migration-plan-cam.adoc[leveloffset=+1]
-include::modules/migration-running-migration-plan-cam.adoc[leveloffset=+1]
+[role="_additional-resources"]
+== Additional resources
+
+* link:https://docs.openshift.com/container-platform/3.11/install_config/registry/securing_and_exposing_registry.html#exposing-the-registry[Exposing a secure registry manually on an {product-title} 3 cluster]
```

Comment:
We can't do hard links like this. Since these docs show up on the portal too, this wouldn't be an appropriate link. Unfortunately, I don't think there's a workaround here, so you won't be able to provide a link for 3.11 here.

Reply:
That's going to be a problem. The user has to expose the secure registry on the 3.11 cluster.

@vikram-redhat Can we make an exception here? It is not easy for the user to find the correct section in the 3.x docs.

Reply:
@bergerhoffer @XinRedhat I have been giving this some thought. Since this is a procedure for migrating from OCP3 to 4, I think we can assume that the OCP3 admin (target audience) knows how to expose a registry manually. So I am going to remove the OCP3 doc link and just leave the OCP4 xref.